python/ray/_raylet.pyx

```diff
@@ -480,42 +480,50 @@ cdef execute_task(
         const c_vector[CObjectID] &c_return_ids,
         c_vector[shared_ptr[CRayObject]] *returns):
 
-    worker = ray.worker.global_worker
-
-    actor_id = worker.core_worker.get_actor_id()
-    job_id = worker.core_worker.get_current_job_id()
-    task_id = worker.core_worker.get_current_task_id()
+    cdef object worker = ray.worker.global_worker
```

Comment:
Interesting, I'm surprised that the cython compiler doesn't do this under the hood anyways

Reply:
The object stuff is done under the hood, I just did it to be more consistent. Reverted it to be more consistent with the rest of `_raylet.pyx`.